core-java-modules/core-java-arrays-operations-advanced/README.md

```diff
@@ -0,0 +1,16 @@
+## Core Java Arrays - Advanced Operations
+
+This module contains articles about advanced operations on arrays in Java
+
+Advanced operations would be much logic or interactions with other arrays, such as:
+
+- Sum, average of elements
+- Intersections
+- ...
+
+### Relevant Articles: 
+- [How to Copy an Array in Java](https://www.baeldung.com/java-array-copy)
+- [Arrays.deepEquals](https://www.baeldung.com/java-arrays-deepequals)
+- [Find Sum and Average in a Java Array](https://www.baeldung.com/java-array-sum-average)
+- [Intersection Between two Integer Arrays](https://www.baeldung.com/java-array-intersection)
+- [[More basic operations-->]](/core-java-modules/core-java-operations-basic)
```

Comment:
Does this wording make sense? I think I'd say "more basic operations" if I was on a "Basic operations" page. Maybe you mean "Basic Operations". Either way, I'd probably recommend that we leave links to just be within the same module set for now, so no link here.